### PR TITLE
srt-live-server: init at 1.4.8

### LINKED ISSUE #119576: packaging request: srt-live-server
**Project description**
> srt-live-server(SLS) is an open source live streaming server for low latency based on Secure Reliable Tranport(SRT).

**Metadata**

* homepage URL: https://github.com/Edward-Wu/srt-live-server
* source URL: https://github.com/Edward-Wu/srt-live-server (git tags are mostly available)
* license: MIT
* platforms: unix
* blockers:
  * https://github.com/Edward-Wu/srt-live-server/issues/93

### PR DESCRIPTION
**Pull request by SCOTT-HAMILTON**
###### Motivation for this change
Closes https://github.com/NixOS/nixpkgs/issues/119576

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119606` at 0b018e65 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>srt-live-server</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Please add a comment on the line above, explaining the purpose of this patch.
Near pkgs/applications/video/srt-live-server/default.nix:19:15:
```
   |
19 |   patches = [ ./fix-insecure-printfs.patch ];
   |               ^
```
</li>
  </ul>
</details>

**Comment by SCOTT-HAMILTON** [comment]
I don't understand, there aren't any trailing whitespace at default.nix:23

File: pkgs/applications/video/srt-live-server/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    repo = "srt-live-server";
+    rev = "V${version}";
+    sha256 = "0x48sxpgxznb1ymx8shw437pcgk76ka5rx0zhn9b3cyi9jlq1yld";
+  };
+
+  patches = [
+    # https://github.com/Edward-Wu/srt-live-server/pull/94
+    ./fix-insecure-printfs.patch
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/applications/video/srt-live-server/default.nix:23]:
```suggestion

```

File: pkgs/applications/video/srt-live-server/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    repo = "srt-live-server";
+    rev = "V${version}";
+    sha256 = "0x48sxpgxznb1ymx8shw437pcgk76ka5rx0zhn9b3cyi9jlq1yld";
+  };
+
+  patches = [
+    # https://github.com/Edward-Wu/srt-live-server/pull/94
+    ./fix-insecure-printfs.patch
+  ];
+  buildInputs = [ srt zlib ];
```
Comment (SuperSandro2000) [inline pkgs/applications/video/srt-live-server/default.nix:23]:
```suggestion
  ];

  buildInputs = [ srt zlib ];
```

**Pull request merged**